base256emoji.go

```diff
@@ -0,0 +1,130 @@
+package multibase
+
+import "strconv"
+
+var base256emojiTable = [256]rune{
+	// Curated list, this is just a list of things that *somwhat* are related to our comunitty
+	'🚀', '🙬', '🙭', '🙮', '🙯', '🪐', '☄', '🛰', '🌌', // Space
```

Comment:
Items 2-5 (inclusive) are not visible on Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/96.0.4664.110 Safari/537.36

Reply:
removed